26.md

```diff
@@ -52,6 +52,8 @@ For example, the following condition strings are valid:
 - `kind=0&kind=1&created_at>1675721813`
 - `kind=1&created_at>1674777689&created_at<1675721813`
 
+It is advisable when constructing a delegation token query string with a `created_at` ***before*** operator, that a `created_at` ***after*** operator is also provided to precisely scope the time window that the delegation token is valid for. This prevents the delegatee from publishing historic notes on the delegator's behalf.
```

Comment:
I think that if adding a `created_at` ***after*** operator is advisable, then it is so regardless of whether a ***before*** operator is present or not.

They achieve different purposes, and the presence of the ***before*** operator has no bearing on whether the ***after*** operator is advisable.

Reply:
This is true, thanks for your feedback - I will update it when I have a moment.

I may change the wording a little also, such that it indicates that for _the vast majority of use-cases_, it is advisable, rather than implying for all usecases.

Reply:
Updated! I'm going to resolve this conversation but feel free to unresolve it or create a new one if there is anything to add.